### ISSUE TITLE
The underlying issue is that net.dns1 and net.dns2 are obsolete and any app that uses them is wrong. The VPNService API since 5.0 or 6.0 does not set these anymore. (There is another close issue about this).

### ISSUE THREAD

**Issue by craigphicks** [schwabe/ics-openvpn#998]
The underlying issue is that net.dns1 and net.dns2 are obsolete and any app that uses them is wrong. The VPNService API since 5.0 or 6.0 does not set these anymore. (There is another close issue about this).

_Originally posted by @schwabe in https://github.com/schwabe/ics-openvpn/issues/575#issuecomment-263228841_

**Comment by schwabe** [comment]
Yes. So what?

**Comment by craigphicks** [comment]
So sorry!  Entirely accidental - didn't even realize this message had been posted.  

In fact I did some testing by monitoring packets and using dnsleaktest - the results indicate it is working correctly.  I posted my results on the comment of [this external post](https://www.codeword.xyz/2016/09/08/solving-openvpn-dns-issues-on-android-clients/) and also the [issue}(https://github.com/schwabe/ics-openvpn/issues/575#issuecomment-460358325) you already closed.

I really want to thank you for your great work.
